aries-site/src/examples/templates/dashboards/components/NotificationItem.js

```diff
@@ -0,0 +1,50 @@
+import PropTypes from 'prop-types';
+import { Anchor, Box, List, Text } from 'grommet';
+
+export const NotificationItem = ({ defaultItemProps, items, itemProps }) => (
```

Comment:
This naming is confusing. I would expect NotificationItem to represent a single thing, but here it is representing a list of things.